### PR TITLE
Add workaround for chromium browsers

### PR DESCRIPTION
**Pull request by ebopalisesy**

### CODE REVIEW

File: intranet/apps/signage/views.py [hunk 1]
```diff
@@ -113,6 +113,10 @@ def touch_signage(request, sign=None, landscape=False):
     if sign and sign.zoom:
         zoom = sign.zoom
 
+    # Chromium has a weird bug where zoom level of 300% is the same as zoom level 100%
+    if zoom == 3:
+        zoom = '301%'
```
Comment (ovkulkarni) [inline intranet/apps/signage/views.py]: how will this work? does signage convert to 300% internally?

Comment (ebopalisesy) [inline intranet/apps/signage/views.py]: The CSS tag takes either a float/int or a percent (3.0 or 3 == 300%)

**Comment by pefoley2** [comment]
Why is this an issue? what versions are impacted? is there an upstream bug?

**Comment by ebopalisesy** [comment]
The underlying issue is that Chromium is not handling CSS precedence correctly.  It could be fixed by changing this or by adding an !important in front of the CSS tag in the template.

**Comment by pefoley2** [comment]
Is there an upstream bug?

**Comment by ebopalisesy** [comment]
Not that I could find, no.

**Comment by pefoley2** [comment]
Then file one and put a link to it in the comment explaining why we're doing this.

**Comment by ebopalisesy** [comment]
deleting old workaround and removing the conflicting css tag seems to be another way to fix the issue

**Pull request merged**